mods/gen/rules/player/ai.yaml

```diff
@@ -535,158 +535,165 @@ Player:
 		PassengerTypes: infantry.hacker, infantry.super_hacker
 
 	SupportPowerBotASModule:
-		RequiresCondition: enable-any-ai
+		RequiresCondition: enable-normal-ai || enable-hard-ai
```

Comment:
Not sure if i like all SWs being disabled to Easy AI like this. Especially for PC/Scud/Nuke, would be weird that AI still builds them but not fire. Making them not build SWs/Not take powerful GPs would be better but we merged the Production Modules.

Reply:
If my memory is correct, in General easy AI won't use support power. 

In addition, it is kinda important for prc to build its super weapon as there are 3 important upgrades need to be brought here, and if easy prc can have super weapon, we should also allow easy USA and gla to have their super weapon.

Reply:
I was thinking, we should still make main logic of general ranking system code in c# and expose script interface to show it on UI, in doing so we can write bot module to let the bot choose properly between General Skill and thus have different strategy.